package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,343 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Principal Component Analysis (PCA) --- :mod:`MDAnalysis.analysis.pca`
+=====================================================================
+
+:Authors: John Detlefs
+:Year: 2016
+:Copyright: GNU Public License v3
+
+This module contains the linear dimensions reduction method Principal Component
+Analysis (PCA). PCA sorts a simulation into 3N directions of descending
+variance, with N being the number of atoms. These directions are called
+the principal components. The dimensions to be analyzed are reduced by only
+looking at a few projections of the first principal components. To run a
+Principal Component Analysis, please refer to the :ref:`PCA-tutorial`.
+
+This module constructs a covariance matrix wherein each element of the matrix
+is denoted by (i,j) row-column coordinates. The (i,j) coordinate reflects the
+influence of the of the ith frame's coordinates on the jth frame's coordinates
+of a given trajectory. The eponymous components are the
+eigenvectors of this matrix.
+
+For each eigenvector, its eigenvalue reflects the variance that the eigenvector
+explains. This value is made into a ratio. Stored in
```

Comment:
`This value is made into a ratio`. That sentence doesn't give any information and is incorrect since it isn't clear what ratio is being build.

Just talk that a cumulated variance is stored to quickly find out how many pca_components are needed to explain x% of the total variance/